File: Chapter 3 python/task12.py
number = int(input("Enter a five-digit integers: "))

first_integer = (number // 10000)
second_integer = (number // 1000) % 10
third_integer = (number // 100) % 10
fourth_integer = (number // 10) % 10
fifth_integer = (number % 10)

if (first_integer == fifth_integer and second_integer == fourth_integer):

	print(number,"is a palindrome")

else:

	print(number,"is not a palindrome")

File: Chapter 3 Java/Task12.java
import java.util.Scanner;
public class Task12 {
       public static void main(String... args) {
       Scanner input = new Scanner(System.in);

       System.out.print("Enter a five-digit integer: ");
       int number = input.nextInt();

       int firstNumber = (number / 10000);
       int secondNumber = (number / 1000) % 10;
       int thirdNumber = (number / 100) % 10;
       int fourthNumber = (number / 10) % 10;
       int fifthNumber = number % 10;

       if (firstNumber == fifthNumber && secondNumber == fourthNumber) {
       System.out.println(number + " is a palindrome");
       }
       else {
       System.out.println(number + " is not a palindrome");
       }

  }
  
}